quinn-proto/src/tests/mod.rs

```diff
@@ -2760,3 +2750,19 @@ fn reject_new_connections() {
     pair.server.assert_no_accept();
     assert!(pair.client.connections.get(&client_ch).unwrap().is_closed());
 }
+
+#[test]
+fn reject_remote_address() {
+    let _guard = subscribe();
+    let mut pair = Pair::default();
+    pair.server.retry_policy = RetryPolicy(Box::new(|incoming| {
+        incoming.remote_address();
```

Comment:
Mostly was making sure it doesn't panic. But overall this test is weird. I removed the test outright because the new test added to `quinn` tests this behavior and is in a better position to do so what with it actually having real IP addresses in a normal way.